### DISCUSSION TITLE
六月更新

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1225] [通知]
**该版本可能不稳定，仅供测试**

花了一个月的时间重写了代码，用依赖注入建立了新的应用架构，总算是在驶向屎山的路上踩了刹车。

重新构建的应用在UI上没有大的变化，但也有一些调整：

- 调整导航方式，二级页面成为一个新的层级
- 播放控制器重写，解决播放结束后应用崩溃的问题
- 提高页面切换体验，降低滚动加载时的卡顿感

这个月的目标已经完成，下个月目标：XBOX，敬请期待。

*由于这个版本未在xbox上测试，所以不会发布到应用商店*

## 🚀 功能

- 添加临时播放记录按钮 @Richasy (#1224)

## 🐛 Bug 修复

- 修复新动态通知的链接问题 @Richasy (#1223)
- 重新搭建应用架构 @Richasy (#1222)
- 过滤切换片源期间非预期的播放错误 @Richasy (#1137)

## 🧰 管理与维护

- 重新搭建应用架构 @Richasy (#1222)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2206.1.0'>六月更新</a>.</em>

**Comment by heartacker** [comment]
大工程啊

**Comment by St-Cai** [comment]
辛苦了 加油

**Comment by SA-Root** [comment]
还以为大佬弃坑了，加油啊

**Comment by EastMG** [comment]


**Reply by EastMG** [reply]
![IMG_20220625_203102.jpg](https://user-images.githubusercontent.com/49364679/175773582-340dcb42-4a34-421d-8612-87f142eae03d.jpg)

**Comment by I-Potato-You** [comment]
辛苦了
